src/main/java/org/taf/utils/RepositoryParser.java

```diff
@@ -0,0 +1,29 @@
+package org.taf.utils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class RepositoryParser {
```

Comment:
Right now this class is intended for reading properties only. Maybe rename to PropertiesReader?

Reply:
It was renamed.